server/src/main/java/org/opensearch/indices/replication/copy/package-info.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Package containing classes to implement a copy of replication */
```

Comment:
Nitpick - could you elaborate on this? I'm not clear on what "implement a copy of replication" means

Reply:
removed this package completely. we moved ReplicationCheckppint class to replication package